test/token-pool-proxy.test.js

```diff
@@ -109,124 +100,97 @@ contract('Token Pool', accounts => {
       const tx = ctx.contracts.pool.setTokenPoolReward(0, {from: ctx.actors.mfg})
       await errors.expectRevert(tx)
     })
-    it("sets new reward", async () => {
-      const tx = await  ctx.contracts.pool.setTokenPoolReward(10, {from: ctx.actors.mfg})
+    it('sets new reward', async () => {
+      const tx = await ctx.contracts.pool.setTokenPoolReward(10, {from: ctx.actors.mfg})
 
       expect(tx.logs.length).to.be.equal(1)
       const log = tx.logs[0]
       expect(log.event).to.be.equal('TokenPoolRewardUpdated')
```

Comment:
Also add assertions to test log value parameters.   Need also query the storage contract to confirm the 10 was actually stored.

Reply:
```
 const tx = await ctx.contracts.atonomi.setTokenPoolReward(newReward, {from: ctx.actors.mfg})
        expect(tx.logs.length).to.be.equal(1)
        expect(tx.logs[0].event).to.be.equal('TokenPoolRewardUpdated')
        expect(tx.logs[0].args._sender).to.be.equal(ctx.actors.mfg)
        expect(tx.logs[0].args._newReward.toString(10)).to.be.equal(newReward.toString(10))

        const pool = await ctx.contracts.atonomi.pools(ctx.actors.mfg)
        expect(pool[0].toString(10)).to.be.equal('0')
expect(pool[1].toString(10)).to.be.equal(newReward.toString(10))
```

Reply:
added log event test. storage test pending figuring out how to pack arguments.

Reply:
using const hashKey = web3Utils.soliditySha3('pools', ctx.actors.mfg,'rewardAmount') worked